Source/Fuse.Scripting.JavaScript/ObservableProperty.uno

```diff
@@ -40,16 +41,95 @@ namespace Fuse.Scripting.JavaScript
 
 		Observable _observable;
 
+		// JS thread
 		internal Observable GetObservable(Scripting.Context context)
 		{
 			if (_observable == null)
 			{
 				_observable = Observable.Create(context, _worker);
+				_observable.Object["_defaultValueCallback"] = (Scripting.Callback)DefaultValueCallback;
 				Subscribe(context);
 			}
 			return _observable;
 		}
 
+		// JS thread
+		object DefaultValueCallback(Scripting.Context context, object[] args)
+		{
+			var value = args[0];
+
+			object marshalledValue;
+			if (!Marshal.TryConvertTo(_property.PropertyType, value, out marshalledValue))
+			{
+				return null;
+			}
+
+			var resolveClosure = new ResolveDefaultValueClosure(this, marshalledValue);
+			UpdateManager.PostAction(resolveClosure.Perform);
+
+			return null;
+		}
+
+		internal class ResolveDefaultValueClosure
+		{
+			readonly Uno.UX.Property _property;
+			readonly ISubscription _subscription;
+			readonly object _value;
+			readonly Action<object> _pushValueOnJSThread;
+
+			public ResolveDefaultValueClosure(ObservableProperty op, object value)
+			{
+				_property = op._property;
+				_subscription = op._subscription;
+				_pushValueOnJSThread = op.PushValueOnJSThread;
+				_value = value;
+			}
+
+			// UI thread
+			public void Perform()
+			{
+				// HACK: https://github.com/fusetools/fuselibs-public/issues/541#issuecomment-335101235
```

Comment:
This seems like something that should be fixed before we merge, no?

Reply:
This is the stuff that we want to delay fixing after release as discussed in the daily last friday

Reply:
Yeah, I figured after seeing this. So I guess this part is OK for now.